paasta_tools/kubernetes/bin/paasta_secrets_sync.py

```diff
@@ -196,14 +207,15 @@ def sync_secrets(
     vault_cluster_config: Mapping[str, str],
     soa_dir: str,
     namespace: str,
+    vault_token_file: str,
```

Comment:
thoughts on having this look something like:
```suggestion
    vault_token_file: str = DEFAULT_VAULT_TOKEN_FILE,
```
this way, no tests need to be updated and the same defaults from using the cli version of this apply to anyone just importing `sync_secrets`